docs/registry/getting-started-registry/README.adoc

```diff
@@ -176,45 +176,56 @@ image::upload-schema.png[Image of form to upload a schema]
 +
 * *Group*: Enter an optional unique group name such as `my-org` to organize the artifact in a named collection. Each group contains a logically related set of schemas or API designs, typically managed by a single entity, belonging to a particular application or organization.
 +
-NOTE:  Specifying a group is optional when using the web console, and a `default` group is automatically created.
+NOTE: Specifying a group is optional when using the web console, and a `default` group is automatically created.
 +
 * *ID*: Enter an optional unique ID for this artifact such as `my-ID`. If you do not specify a unique artifact ID, {registry} generates one automatically as a UUID.
-* *Type*: Use the default *Auto-Detect* setting to automatically detect the artifact type, or select the artifact type from the drop-down, for example, Avro Schema or OpenAPI.
+* *Type*: Use the default *Auto-Detect* setting to automatically detect the artifact type, or select the artifact type from the list, for example, Avro Schema or OpenAPI.
 * *Artifact*: Drag and drop or click *Browse* to upload a file. For this example, copy and paste the following simple Avro schema:
 +
 [source,json,subs="+quotes,attributes"]
 ----
 {
-"type": "record",
-"namespace": "com.example",
-"name": "FullName",
-"fields": [
-{ "name": "first", "type": "string" },
-{ "name": "last", "type": "string" }
-]}
+  "type": "record",
+  "namespace": "com.example",
+  "name": "FullName",
+  "fields": [
+    {
+      "name": "first",
+      "type": "string"
+    },
+    {
+      "name": "last",
+      "type": "string"
+    }
+  ]
+}
 ----
 
 . Click *Upload* to complete the operation and display the new artifact details:
 
-* *Info*: Displays the artifact name, group, description, lifecycle status, when created, and last modified.
-* *Content*: Displays a read-only view of the full artifact content.
-* *Documentation*: (OpenAPI only): Displays automatically-generated REST API documentation.
-* *Content Rules*: Displays artifact content rules that you can enable and configure. You can configure a *Validity Rule* or *Compatibility Rule* by selecting the appropriate rule configuration from the drop-down. For details on supported rules, see the https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1[{product-long-registry} user documentation^].
+* *Overview* tab:
+** *Version metadata*: Displays details such as the artifact name, artifact ID, description, lifecycle status, when created, owner, last modified, global ID, content ID, labels, and properties.
```

Comment:
Don't think we need to list all metadata details. When I read "details such as...", I'm expecting a few examples, not a comprehensive list of what I can see by looking at the UI? 

Not a huge deal either way, but don't think listing everything adds value in terms of minimal docs. 

For example, suggest something like `...details such as the artifact name, artifact ID, global ID, content ID, labels, properties, and so on`.

Reply:
@smccarthy-ie Done :)